bundles/org.openhab.binding.enturno/src/main/java/org/openhab/binding/enturno/internal/model/Data.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.enturno.internal.model;
+
+import org.openhab.binding.enturno.internal.model.stopplace.StopPlace;
+
+/**
+ * Generated Plain Old Java Objects class for {@link Data} from JSON.
+ *
+ * @author Michal Kloc - Initial contribution
+ */
+public class Data
```

Comment:
Bracket should be on the same line.
```suggestion
public class Data {
```